config/environments/development.rb

```diff
@@ -42,9 +42,9 @@
   config.paperclip_defaults = {
     :storage => :s3,
     :s3_credentials => {
-      :bucket => ENV['mythicaleatures'],
-      :access_key_id => ENV['AKIAJXIU2NTHSKIVAN5Q'],
-      :secret_access_key => ENV['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']
+      :bucket => ENV['MY_BUCKET_NAME'],
+      :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.